examples/README.md

```diff
@@ -39,3 +39,22 @@ improvements. Details are discussed
 	  dedicated project called
 [MesaPy](https://github.com/mesalock-linux/mesapy). In this specific example,
 we demonstrate how to invoke the MesaPy engine integrated into MesaTEE.
+* [DBSCAN](DBSCAN)
+	- Provides an implementaton of DBSCAN clustering.
+* [Generalized Linear Model](GeneralizedLinearModel)
+	- Contains implemention of generalized linear models using iteratively reweighted least squares.The model will automatically add the intercept term to the input data.
```

Comment:
I suggest to break the line by 80 characters. This is more friendly for diff in version control system like Git.

Reply:
Same for other lines in the `README.md`.